src/topp/OpenSwathDecoyGenerator.cpp

```diff
@@ -280,12 +280,12 @@ class TOPPOpenSwathDecoyGenerator
       if (separate)
       {
         OPENMS_LOG_INFO << "Writing only decoys to file: " << out << std::endl;
-        targeted_merged = std::move(targeted_decoy);
+        targeted_merged = targeted_decoy;
```

Comment:
not sure why move removed - check
also below

Reply:
I guess because move assignment operator is missing?

https://github.com/OpenMS/OpenMS/blob/8594c2ad50b1e3d3263235ea592ed02049dbcb11/src/openms/include/OpenMS/ANALYSIS/TARGETED/TargetedExperiment.h#L103

Reply:
done in https://github.com/OpenMS/OpenMS/pull/5686/files